lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -392,9 +392,8 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
                 width: 24, height: 24, fit: BoxFit.fill),
             AppLocalizations.of(context).download_to_device,
             document)
-        .onActionClick((data) {
-      mySpaceViewModel.downloadFile(data.documentId);
-    }).build();
+        .onActionClick((data) => getIt<Store<AppState>>().dispatch(mySpaceViewModel.handleDownloadFile(data.documentId)))
```

Comment:
Why don't we just call `handleDownloadFile()`. Then `handleDownloadFile()` will handle that clicked, then do some sequence actions like permission, download, dispatch. We also can access store in model instead of `getIt`

Reply:
We want to dispatch a thunk to store to execute an asynchronous task, do you have better choice to me?

Reply:
Why don't you make a function in viewModel that call an action instead of putting it there ?